src/test/java/spleetwaise/transaction/logic/parser/ParserUtilTest.java

```diff
@@ -99,4 +101,28 @@ public void getPersonFromPhone_personFound_success() {
         Person testPerson = assertDoesNotThrow(() -> ParserUtil.getPersonFromPhone(TEST_PHONE));
         assertEquals(TypicalPersons.ALICE, testPerson);
     }
+
+
+    @Test
+    public void getPersonFromIndex_null_exceptionThrown() {
+        assertThrows(NullPointerException.class, () -> ParserUtil.getPersonFromAddressBookIndex(null));
+    }
+
+    @Test
+    public void getPersonFromIndex_personNotFound_exceptionThrown() {
+        AddressBookModel abModel = new AddressBookModelManager();
+        CommonModel.initialise(abModel, null);
+
+        assertThrows(ParseException.class, () -> ParserUtil.getPersonFromAddressBookIndex(TEST_INDEX));
+    }
+
+    @Test
+    public void getPersonFromIndex_personFound_success() {
+        AddressBookModel abModel = new AddressBookModelManager();
+        CommonModel.initialise(abModel, null);
+        abModel.addPerson(TypicalPersons.ALICE);
+
+        Person testPerson = assertDoesNotThrow(() -> ParserUtil.getPersonFromAddressBookIndex(TEST_INDEX));
+        assertEquals(TypicalPersons.ALICE, testPerson);
+    }
```

Comment:
There should be additional test cases here to get the Person from a filteredList (Happens when a user uses the `find` command). Check for personFound and personNotFound